CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
-v1.5 / 2012-12-11
+v1.6/ 2020-01-15
+========================
+
+* fix cleanup mempool related thread while destroying a volume ([#303](https://github.com/openebs/istgt/pull/303),
```

Comment:
now onwards, lets keep the commit header (for e.g. fix(crash): ) in the change log.

Reply:
Done